app/src/main/java/com/example/giphyapplication/ui/features/gifdetails/GifDetailsViewState.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.giphyapplication.ui.features.gifdetails
+
+sealed class GifDetailsViewState {
+    object Loading : GifDetailsViewState()
+    object Error : GifDetailsViewState()
+    object Success : GifDetailsViewState()
```

Comment:
The success view state should hold your data and here you should add the gif property.